guides/doc-Release_Notes/topics/katello-4.8.0.adoc

```diff
@@ -0,0 +1,170 @@
+= 4.8.0 Phoenix (2023-03-28)
+
+== Features
+
+=== Hosts
+ * Add support for Erratum release date in Host - Applied Errata report template (https://projects.theforeman.org/issues/36049[#36049], https://github.com/Katello/katello/commit/d1b4c553b19d3056dc5ca5cb133e880b5
+1df638f[d1b4c553])
```

Comment:
There are also quite a few line breaks in this file in the middle of links. That trips up the link checker.